docs/socket-types.md

```diff
@@ -0,0 +1,16 @@
+this.socket.on('connect', this.onSocketConnected());
+    this.socket.on('connect_error', this.onSocketConnectionError);
+    this.socket.on('connect_timeout', this.onSocketTimeout);
+    this.socket.on('disconnect', this.onSocketDisconnected());
+    this.socket.on('reconnect', this.onSocketReconnected());
+    this.socket.on('reconnecting', this.onSocketReconnecting);
+    // this.socket.on('reconnect_error', socketConnectionError);
+    this.socket.on('reconnect_failed', this.onSocketReconnectionFailed);
+    this.socket.on('reconnect_timeout', this.onSocketTimeout);
```

Comment:
I guess future work would involve updating the message (not the background colour) for these scenarios